theme/less/_variables.less

```diff
@@ -1,4 +1,6 @@
 // Colors
-@primary: #728448;
-@secondary: #52544e;
+@primary: #313131;
```

Comment:
`#313131` seem to be outside of `#4d4d4d` and `#616161` which according to the color picker in chrome devtools are colors of Logo text and navigation item accordingly.

Reference comment: https://github.com/Privacy-Managers/privacy-manager-website/pull/5#discussion_r471462033

Reply:
But I think you can use `#4d4d4d` as it's darker than `#616161`, but still doesn't make the navigation text look too diferent.

Reply:
Done. But anyway #4d4d4d looks lighter to me on the website then #313131.

Reply:
Yes, you mean it less accessible because of that?

I suggested it as it was closer to what we have in Figma.